### DISCUSSION TITLE
Outdated code shown in compare/pull request

### DISCUSSION THREAD

**Discussion by Drift91** [community/community#21902] [New to GitHub]
<p>I’ve forked someone’s repo to customize it for my own use. Since then they’ve updated their code and I want to merge the latest commits of thier master into my customized fork to keep my version up-to-date yet still customized.</p>
<p><a href="https://github.com/Drift91/Cops_FiveM/compare/dev...Kyominii:master" rel="nofollow noopener">https://github.com/Drift91/Cops_FiveM/compare/dev…Kyominii:master</a></p>
<p>Now if you compare the above comparison of changelog.md to the actual contents below, you’ll see that the comparison above is completely out of date.</p>
<p><a href="https://github.com/Drift91/Cops_FiveM/blob/dev/CHANGELOG.md" rel="nofollow noopener">https://github.com/Drift91/Cops_FiveM/blob/dev/CHANGELOG.md</a></p>
<p>It seems as if it’s comparing <em>his</em> master <em>before I forked</em> it to his current master instead of comparing <em>my</em> <em>current</em> dev branch to his current master branch. Is this a bug in the website or am I doing something wrong? If this is how it’s going to function then making a pull request to my code is useless and I might as well do everything manually in a text editor.</p>
<p>Does someone know how to properly compare the most recent code of both forks? I’ve even tried using the hash of my most recent commit to compare and it still shows the outdated code. What’s weird is that even if I switch the direction of the comparison, it still shows the same outdated 1.4.0 code on the left.</p>
<p>Thanks in advance for any assistance.</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/drift91">@drift91</a></p>
<p>I’d recommend taking a look at our Help Docs on how to merge an upstream repository to your fork:</p>
<p><a href="https://help.github.com/articles/merging-an-upstream-repository-into-your-fork/" rel="nofollow noopener">https://help.github.com/articles/merging-an-upstream-repository-into-your-fork/</a></p>
<p>If there are conflicts (which it sounds like there might be), you’ll also want to check out the article on addressing merge conflicts:</p>
<p><a href="https://help.github.com/articles/addressing-merge-conflicts/" rel="nofollow noopener">https://help.github.com/articles/addressing-merge-conflicts/</a></p>
<p>I hope these help! If you have further questions, please let me know!</p>

**Comment by valeriakononenko** [comment]
<p>I have the same problem. I created new branch, did the changes and tryed to compare branches. Now I have the filein 2 branches, but having differences when I comparing branches. </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/5/548f4f9d65c57b1c0ce863e82626dd62bb8629f0.png" data-download-href="/uploads/short-url/c439ifBEhrCNar7dhM5mmatBSlq.png?dl=1" title="Screenshot from 2018-03-14 14-33-38.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/3019034/181067477-fcc0b1ba-9148-4436-87a7-f9402ec3e92f.png" alt="Screenshot from 2018-03-14 14-33-38.png" data-base62-sha1="c439ifBEhrCNar7dhM5mmatBSlq" width="690" height="387" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/5/548f4f9d65c57b1c0ce863e82626dd62bb8629f0_2_690x387.png, https://aws1.discourse-cdn.com/github/optimized/2X/5/548f4f9d65c57b1c0ce863e82626dd62bb8629f0_2_1035x580.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/5/548f4f9d65c57b1c0ce863e82626dd62bb8629f0.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/5/548f4f9d65c57b1c0ce863e82626dd62bb8629f0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot from 2018-03-14 14-33-38.png</span><span class="informations">1366×768 83.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> <div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/e/e9f430810478be058d0752d05c5c1e7d6aa25dc7.png" data-download-href="/uploads/short-url/xnEqio2WWSlSqg7s86OydXyCWt9.png?dl=1" title="Screenshot from 2018-03-14 14-33-40.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/3019034/181067478-6cd93f92-1eec-4675-921e-20054a89120b.png" alt="Screenshot from 2018-03-14 14-33-40.png" data-base62-sha1="xnEqio2WWSlSqg7s86OydXyCWt9" width="690" height="387" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/e/e9f430810478be058d0752d05c5c1e7d6aa25dc7_2_690x387.png, https://aws1.discourse-cdn.com/github/optimized/2X/e/e9f430810478be058d0752d05c5c1e7d6aa25dc7_2_1035x580.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/e/e9f430810478be058d0752d05c5c1e7d6aa25dc7.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/e/e9f430810478be058d0752d05c5c1e7d6aa25dc7_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot from 2018-03-14 14-33-40.png</span><span class="informations">1366×768 83.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> <div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/4/40ff111af6a9c7745d5664c55c9b145313d5a6c0.png" data-download-href="/uploads/short-url/9gZ2fkgAuS9OKXgJFGoNTsBoRvq.png?dl=1" title="Screenshot from 2018-03-14 14-33-58.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/3019034/181067479-988de70c-ea15-43dd-9b6b-a995ad4784a6.png" alt="Screenshot from 2018-03-14 14-33-58.png" data-base62-sha1="9gZ2fkgAuS9OKXgJFGoNTsBoRvq" width="690" height="387" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/4/40ff111af6a9c7745d5664c55c9b145313d5a6c0_2_690x387.png, https://aws1.discourse-cdn.com/github/optimized/2X/4/40ff111af6a9c7745d5664c55c9b145313d5a6c0_2_1035x580.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/4/40ff111af6a9c7745d5664c55c9b145313d5a6c0.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/4/40ff111af6a9c7745d5664c55c9b145313d5a6c0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot from 2018-03-14 14-33-58.png</span><span class="informations">1366×768 86.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by lee-dohm** [comment]
<p>What you’re describing <a class="mention" href="/divergence082">@divergence082</a> isn’t the same thing. You’re describing the same problem as is mentioned and answered here:</p>
<p><a href="https://github.community/t/branch-to-branch-compare-compares-to-master-instead/709">https://github.community/t5/How-to-use-Git-and-GitHub/Branch-to-Branch-Compare-Compares-to-Master-Instead/m-p/4677</a></p>
